spec/rails_helper.rb

```diff
@@ -110,7 +161,7 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
-
+  config.include FactoryBot::Syntax::Methods
```

Comment:
I think this line is un-needed if you have factory bot rails isn't it?

Reply:
Without this, I have to specify FactoryBot instead of just calling create within the rspec let blocks.

Reply:
gotcha